src/init/global.init.s

```diff
@@ -4,20 +4,34 @@
 global_init:
 	ld hl,global_.sheets
 	ld b,n_sheets
-@sheets_loop:
-;	for now they are all rocks
+
+; ////// ROCK \\\\\\
 	ld a,%10010000	; size = 0, block, hurt by bombs.
 	ldi (hl),a
 	ld a,0	; dmg = 0
 	ldi (hl),a
 	ld a,0
 	ldi (hl),a
 	ldi (hl),a ; no funtion pointer or whatever
+	ldi (hl),a ; speed = 0
+	ld a,0

```

Comment:
Why ? a is already 0

Reply:
It's easier to modify (it the element HP, set it to 1 add the element in dysplay_init.s)
It's an init file, performance isn't an issue.